onnxruntime/hosting/http/routes.cc

```diff
@@ -47,7 +47,6 @@ http::status Routes::ParseUrl(http::verb method,
   }
 
   if (func_table.empty()) {
-    std::cout << "Unsupported method: [" << method << "]" << std::endl;
```

Comment:
I would rather keep the log with request id in it. It will be helpful for debugging.

Reply:
It will be done with an error callback.